cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -40,6 +40,7 @@ export interface ActiveElement {
 export interface DrawData {
     enabled: boolean;
     shapeType?: string;
+    boxDrawingType?: string;
```

Comment:
How about renaming it everywhere to ``rectDrawingMethod``?

Reply:
Please, update the interface in ``cvat-canvas/README.md``

Reply:
> How about renaming it everywhere to `rectDrawingMethod`?

Cool. This seems to be more straightforward. Renamed it everywhere.

> Please, update the interface in `cvat-canvas/README.md`

Updated [cvat-canvas/README.md](https://github.com/opencv/cvat/pull/1127/files#diff-80bfdbe0c80ffbc37c31dcaadfa68348R43)

Reply:
BTW, I rebased from the current develop branch to avoid merge-conflict. `Outdated` tag is attached in this thread because of it.